docs/Language/Python/Tries.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Tries
+nav_order: 1
+parent: Python
```

Comment:
As we are nesting the Tires Content from Language we also need to define " grand_parent: Language " as grandParent, parent, Child. Where in this instance Tries is the child of the parent and parent is the child of grandparent